src/app/containers/Rewards/providers/getVaultRewardsAPR.ts

```diff
@@ -0,0 +1,189 @@
+import {
+  AxialLPData,
+  ExtraTokens,
+  MasterchefApr,
+  PoolInfos,
+  PoolTypes,
+} from "../types";
+import axios from "axios";
+import { BigNumber, ethers } from "ethers";
+import { rpcUrl } from "app/containers/BlockChain/utils/wallet/connectors";
+import { pools } from "app/pools";
+import lpAMM from "abi/lpTokenAMM.json";
+import { AXIAL_MASTERCHEF_CONTRACT_ADDRESS, ZERO_ADDRESS } from "../constants";
+import masterchefABI from "abi/masterchef.json";
+import swap from "abi/swapFlashLoanNoWithdrawFee.json";
+import erc20 from "abi/erc20.json";
+import simplerewarder from "abi/simplerewarder.json";
+
+export async function getAVAXPrice(): Promise<number> {
+  const query = JSON.stringify({
+    query: `{ bundle(id:1){ ethPrice } }`,
+    variables: {},
+  });
+
+  const config = {
+    headers: {
+      "Content-Type": "application/json",
+    },
+  };
+
+  try {
+    const response = await axios.post(
+      "https://api.thegraph.com/subgraphs/name/dasconnor/pangolin-dex",
+      query,
+      config
+    );
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    const AVAXPrice = response.data.data.bundle.ethPrice as number;
```

Comment:
if we want to void a bug this call be written a bit different to account for undefined values e.g 
```suggestion
    const AVAXPrice = <number>response.data.data.bundle.ethPrice;
    if (!AVAXPrice) do something.
```

Reply:
lets try not to fight with the linter but instead make it work. The reason why the linter is barking is because this is actually an unsafe access, in addition you could use optional chaining to be ever more safe.